gallery/src/pages/Home.tsx

```diff
@@ -24,35 +27,157 @@ const Home = () =>
+          <NxDescriptionList.Description>
+            This section includes components that the user would activate in order to initiate some action – buttons,
+            dropdowns and other menus, and also links.
+          </NxDescriptionList.Description>
+        </NxDescriptionList.Item>
+        <NxDescriptionList.Item>
+          <NxDescriptionList.Term>Forms</NxDescriptionList.Term>
+          <NxDescriptionList.Description>
+            This sections includes all components that are typically used within a form. Components for forms themselves,
+            form structures such as fieldset, and for various kinds of form fields all fall within this section.
+          </NxDescriptionList.Description>
+        </NxDescriptionList.Item>
+        <NxDescriptionList.Item>
+          <NxDescriptionList.Term>Data Presentation</NxDescriptionList.Term>
+          <NxDescriptionList.Description>
+            This section groups together components which are intended for organizing and displaying data. This includes
+            views such as lists, tables, and trees, and also visualizations such as the donut chart.
+          </NxDescriptionList.Description>
+        </NxDescriptionList.Item>
+        <NxDescriptionList.Item>
+          <NxDescriptionList.Term>Layout</NxDescriptionList.Term>
+          <NxDescriptionList.Description>
+            Components which make up the "page skeleton" are included in this section. This includes page-level components
+            such as the global sidebar and also containers such as tiles and accordions. In addition, rich examples of
+            layouts incorporating multiple components working together are also found here.
+          </NxDescriptionList.Description>
+        </NxDescriptionList.Item>
+        <NxDescriptionList.Item>
+          <NxDescriptionList.Term>Typography and Icons</NxDescriptionList.Term>
+          <NxDescriptionList.Description>
+            This section includes semantic and typographic containers for text content, such as paragraphs and code
+            blocks.  It also includes utility components that facilitate the use of icons.
+          </NxDescriptionList.Description>
+        </NxDescriptionList.Item>
+        <NxDescriptionList.Item>
+          <NxDescriptionList.Term>HTML Variants</NxDescriptionList.Term>
+          <NxDescriptionList.Description>
+            Some RSC components and styles have separate support for usage via React component and via bare CSS classes.
+            In these cases, the React components and bare CSS classes are documented separately. For any such component
+            which has both types of documentation, the HTML documentation is currently sequestered in this section.
+            Generally, for React-using consumers, the React components should be preferred over the bare CSS class
+            approach, and this section should thus generally be ignored.
```

Comment:
There are separate usage guidelines and documentation for some RSC components, depending on whether they are React components or bare CSS classes. For components that have both React and HTML/CSS documentation, React guidelines should be preferred and this section should be ignored.